src/components/button/_button.scss

```diff
@@ -224,20 +224,24 @@
     @include govuk-typography-weight-bold;
     @include govuk-typography-responsive($size: 24, $override-line-height: 1);
 
+    display: inline-flex;
     min-height: auto;
+
     padding-top: govuk-spacing(2) - $govuk-border-width-form-element;
-    padding-right: govuk-spacing(7);
+    padding-right: govuk-spacing(3);
     padding-bottom: govuk-spacing(2) - $govuk-border-width-form-element;
     padding-left: govuk-spacing(3);
 
-    background-image: govuk-image-url("icon-pointer.png");
-    background-repeat: no-repeat;
-    background-position: 100% 50%;
+    justify-content: center;
+  }
 
-    @include govuk-device-pixel-ratio {
-      background-image: govuk-image-url("icon-pointer-2x.png");
-      background-size: 30px 19px;
-    }
+  .govuk-button__start-icon {
+    margin-top: -.125em;
+    margin-right: -.08333333333333333em;
+    margin-left: .4166666666666667em;
```

Comment:
Could these use `govuk-em`?

Reply:
When I initially did this the icon scaled alongside the font size, but to keep it consistent with the current implementation it no longer does that.

So with that in mind we don't need to use relative units (em), so I've changed it back to pixels.

Reply:
I think this should be:

```
margin-top: -3px;
margin-right: 0; (or not there)
margin-left: govuk-spacing(1);
@include govuk-media-query($from: desktop) {
    margin-left: govuk-spacing(2);
}
```
The spacing scale does not change below `govuk-spacing(4)` so this change from 1 to 2 would need to be done with media queries